### DISCUSSION TITLE
color_scheme of `wezterm connect` window

### DISCUSSION THREAD

**Discussion by ho-oto** [wezterm/wezterm#2081] [Q&A] [answered]
Now, my color_scheme setting does not propagate to the window launched by `wezterm connect` command.
How can I set color_scheme for the window?

version:
wezterm 20220408-101518-b908e2dd on MacOS

wezterm.lua:
https://github.com/ho-oto/dotfiles-for-mac/blob/master/config/xdg/wezterm/wezterm.lua

**Comment by wez** [comment, marked as answer]
The color scheme is defined on the server side when using multiplexing! See: https://github.com/wez/wezterm/issues/1268

**Reply by bew** [reply]
Why is that so anyway? isn't this 'client' configuration? (the keybindings, the colors)

**Reply by wez** [reply]
Explained in https://github.com/wez/wezterm/issues/1268#issuecomment-951562027
